codersdk/workspaceagents.go

```diff
@@ -133,6 +133,16 @@ type WorkspaceAgentMetadata struct {
 	Description WorkspaceAgentMetadataDescription `json:"description"`
 }
 
+type DisplayApp string
+
+const (
+	DisplayAppVSCodeDesktop  DisplayApp = "vscode"
```

Comment:
QQ: do we need to take any action on the API level based these enum values? Maybe `codersdk`/API can be just a "dumb proxy", I mean "any" app.

Reply:
No not really it's mainly for discoverability and it also helps with the generated frontend types but I also don't mind removing it, wdyt?

Reply:
My idea is to reduce the number of places to update when you add another application, so if we don't intend to use let's just drop it. Unless we will use it in TypeScript/site, then we can leave it 👍